Lib/asyncio/base_events.py

```diff
@@ -630,24 +629,23 @@ def set_default_executor(self, executor):
         self._default_executor = executor
 
     def _getaddrinfo_debug(self, host, port, family, type, proto, flags):
-        msg = ["%s:%r" % (host, port)]
+        msg = [f"{host}:{port!r}"]
         if family:
-            msg.append('family=%r' % family)
+            msg.append(f'family={family!r}' % family)
```

Comment:
There's a remaining % family after the f-string

Reply:
Good catch, created a PR to fix this: https://github.com/python/cpython/pull/4787

Thanks!